rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
+- Start Date: 09 February 2018
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Begin using an RFC process. Initially use a structure similar to the process used by the maintainers of IPFS lang, as described in https://github.com/IPFS-lang/rfcs
+
+# Motivation
+[motivation]: #motivation
+
+Establish a clear, repeatable process for people to propose ideas and make decisions in a way that
+1. Allows everyone to see what decisions have been made and how they were made
+2. Allows everyone to see what proposals have been proposed and comment on them or ssuggest changes
+3. Works for a globally distributed group of contributors to discuss and make decisions asynchronously
+4. Can be replicated/repeated in sibling projects or spinoff projects like libp2p, IPLD, etc.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+_[Cribbed directly from [IPFS-lang/rfcs README](https://github.com/IPFS-lang/rfcs#IPFS-rfcs)]_
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the IPFS community and the sub-teams.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the language and standard libraries, so that all stakeholders can be confident about the direction the language is evolving in.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The proposed RFC process is described in the [README](../README.md). It's a modified version of  the process used in the [IPFS lang RFCs repository](https://github.com/IPFS-lang/rfcs/blob/752a02115e49c114e2d6b5247c410da69aac505c/README.md), which is dual-licensed under MIT and Apache2 licenses.
```

Comment:
Same note as up on line 9 — is this supposed to be the “IPFS RFCs repository” and link to `ipfs/rfcs/xyz/README.md` or to “Rust Lang RFCs repository” and link to `rust-lang/rfcs/xyz/README.md`? It’s halfway between those now, so doesn’t seem correct no matter what.